docs/source/noise/index.rst

```diff
@@ -177,10 +177,11 @@ Noise types for each column
     - Decennial Census, ACS, CPS, WIC
     - Leave a field blank, choose the wrong option
     -
-  * - SSN
-    - W-2 and 1099, SSA
+  * - SSN or ITIN
```

Comment:
Do we need to update this on other pages where we reference SSN?

Reply:
I think this cell should still just say "SSN" because that's the name of the column -- none of the datasets have a column called "ITIN". It's just that in the 1040, some of the values in the SSN column will actually be ITINs instead of SSNs, but that's not particularly relevant to the noise functions.